.travis.yml

```diff
@@ -70,7 +70,8 @@ matrix:
         # Needs python prefix to use the correct interpretor.
         - python ./obs_clone.py --cache --debug --apiurl-target local
       script:
-        - nosetests --with-coverage --cover-package=. --cover-inclusive
+        # Ignore the Python3 docker_*.py scripts
+        - nosetests --with-coverage --cover-package=. --cover-inclusive --ignore-files 'docker_.+\.py' --ignore-files 'docker_.+\.py'
```

Comment:
If I remember correctly, there is a config format for nose. If this can be placed there (after seeing that it works) that would be preferred. One place to manage and local runs during development won't require flags.

Reply:
I'll try that - hopefully it supports multiple mentions, which seems counter-intuitive for ini-style files

Reply:
Seems to have resolved the failure on travis although you have packaging fun now :)